core/src/test/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionSpec.scala

```diff
@@ -7,9 +7,19 @@ import akka.testkit.TestKit
 import com.google.api.client.auth.oauth2.Credential
 import org.broadinstitute.dsde.rawls.config.MethodRepoConfig
 import org.broadinstitute.dsde.rawls.dataaccess.{drs, _}
-import org.broadinstitute.dsde.rawls.dataaccess.drs.{DrsHubMinimalResponse, DrsHubResolver, ServiceAccountEmail, ServiceAccountPayload}
```

Comment:
This is all scalafmt, not sure how it got missed by the global reformat last month.